testing/asserts.ts

```diff
@@ -16,6 +16,16 @@ export class AssertionError extends Error {
 export function equal(c: unknown, d: unknown): boolean {
   const seen = new Map();
   return (function compare(a: unknown, b: unknown) {
+    // Have to render RegExp & Date for string comparison
```

Comment:
Can you add a test to testing/asserts_test.ts too?

Reply:
Will add!

Reply:
done